### PR TITLE
feat(components): SageMaker V2 model monitor component release

### PR DESCRIPTION
**Pull request by rd-pong**
**Description of your changes:**

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Review by ryansteakley — APPROVED** [review]

File: components/aws/sagemaker/Changelog.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ The version of the AWS SageMaker Components is determined by the docker image ta
 Repository: [Public ECR](https://gallery.ecr.aws/kubeflow-on-aws/aws-sagemaker-kfp-components) or [Dockerhub](https://hub.docker.com/repository/docker/amazon/aws-sagemaker-kfp-components). New releases after v1.1.1 will be using the public ECR repository
 
 ---------------------------------------------
+** Change log for version 2.3.0 **
+
+- Introducing SageMaker Model Monitor components v2. This release includes [Monitoring Schedule](./MonitoringSchedule/), [DataQualityJobDefinition](./DataQualityJobDefinition/), [ModelQualityJobDefinition](./ModelQualityJobDefinition/), [ModelExplainabilityJobDefinition](./ModelExplainabilityJobDefinition/), [ModelBiasJobDefinition](./ModelBiasJobDefinition/).
+
+> Pull request : [#9253](https://github.com/kubeflow/pipelines/pull/9253), [#9368](https://github.com/kubeflow/pipelines/pull/9368)
```
Comment (surajkota) [inline components/aws/sagemaker/Changelog.md:11]: add a note to not use the commit from this PR #9253 

something along lines of - https://github.com/kubeflow/pipelines/pull/9253#issuecomment-1540804814

**Review by surajkota — APPROVED** [review]
/approve with minor comment

File: components/aws/sagemaker/Changelog.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ The version of the AWS SageMaker Components is determined by the docker image ta
...
+
+This PR [#9253](https://github.com/kubeflow/pipelines/pull/9253) was merged accidentally without the changelog and image pointing to an unofficial repository. Please do not use this commit.
```
Comment (surajkota) [inline components/aws/sagemaker/Changelog.md:13]: **Note:** [#9253](https://github.com/kubeflow/pipelines/pull/9253) was merged accidentally with image pointing to an unofficial repository. Please do not pin to the commit corresponding to this PR.

**Review by ryansteakley — COMMENTED** [review]
/lgtm

**Pull request merged**